wordcount.c

```diff
@@ -0,0 +1,67 @@
+/*Rebecca Hale Assignment 8 Part 1*/
```

Comment:
nice job overall, but your program didn't take all kinds of white spaces into account (tabs, newlines, etc), so it returned a number much lower for the test file.